force-app/tests/Collection Recipes/IterationRecipes_Tests.cls

```diff
@@ -56,13 +56,14 @@ public class IterationRecipes_Tests {
         try {
             Test.startTest();
             IterationRecipes.iterableApiClientRecipe();
-            System.assert(false, 'Expected IterableApiException');
+            System.Assert.fail('Expected IterableApiException');
         } catch (Exception baseException) {
             // Ensure that we caught our custom IterableApiException exception.
             // The catch clause is voluntarily kept generic with Exception as other exceptions could be triggered.
             // We use the instanceof operator to check that baseException is an instance of IterableApiException.
-            System.assert(
-                baseException instanceof IterableApiClient.IterableApiException,
+            System.Assert.isInstanceOfType(
```

Comment:
Fix param order

Reply:
@pozil This one actually flipped me too. If you look at the docs it’s actually other way round compared to other assert methods.

I initially had it reversed only but the build failed and when I looked closely at the docs its first parameter is instance and then expected type. Further details below 

https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_class_System_Assert.htm#apex_System_Assert_isInstanceOfType